python/ray/serve/constants.py

```diff
@@ -37,3 +37,15 @@
 
 #: Name of backend reconfiguration method implemented by user.
 BACKEND_RECONFIGURE_METHOD = "reconfigure"
+
+#: Long poll key for replica handles.
+LONG_POLL_KEY_REPLICA_HANDLES = "replica_handles"
+
+#: Long poll key for traffic policies.
+LONG_POLL_KEY_TRAFFIC_POLICIES = "traffic_policies"
+
+#: Long poll key for backend configs.
+LONG_POLL_KEY_BACKEND_CONFIGS = "backend_configs"
+
+#: Long poll key for route table.
+LONG_POLL_KEY_ROUTE_TABLE = "route_table"
```

Comment:
Can we use enum for this? Group them in `enum` is definitely more maintainable for future than prefix everything with LONG_POLL_

Reply:
good idea